### ISSUE TITLE
[dyld]: Missing symbol called at execution time

### ISSUE THREAD

**Issue by jorge3pp** [airbnb/lottie-ios#1971]
## Which Version of Lottie are you using?
Lottie 4.1.3

## Expected Behavior
The LottieAnimationView is shown and played

## Actual Behavior
When the LottieAnimationView is launched inside a xcframework that another app is using, an error appears at execution time:
`[dyld]: Missing symbol called`

## More info
I have checked and this doesn't happen on previous versions (4.1.2).
While doing a diff
https://github.com/airbnb/lottie-ios/compare/4.1.2...4.1.3
we can see a few changes. 

Between all those commits the only one that seems related seems to be this one:
https://github.com/airbnb/lottie-ios/commit/4247a7a57de59d5ca4922c38007d7a2355d6941a

Right now I am working on providing an example. Sorry for the inconvenience

**Comment by calda** [comment]
When you have a chance can you share a sample project that reproduces this issue?

**Comment by jorge3pp** [comment]
Hey @calda,

I've found out the reason and it's not a bug. Just a mistake on my side.
I was using lottie inside my xcframework. I guess it got cached to previous version of lottie when I tried to update my xcframework.

Then the dependency was downloaded in the app with v4.1.3. So when tried to launch the animation during runtime, it shows an error because there is a mismatch between the version used in the xc and the one downloaded by the app.

I guess I only got this error in 4.1.3 because it had a change in the class I was using. Probably it didn't mattered that there was a mismatch until a class with an actual change.

---
Sorry for the inconvenience.
Thanks for the work you are doing.
